site/content/resources/upgrading.md

```diff
@@ -9,9 +9,68 @@ This document describes the changes needed to upgrade your Contour installation.
 
 <div id="toc" class="navigation"></div>
 
-## Upgrading Contour 1.17.0 to 1.17.1
+## Upgrading Contour 1.17.1 to 1.18.0
```

Comment:
Should we have a big call out about the externalName thing just so folks are aware that upgrading will disable all those unless the config option is set?

Reply:
good idea, I'll add a link to the CVE page